pages/index.tsx

```diff
@@ -1,19 +1,36 @@
 import { Box, Button, Container, Typography } from '@mui/material';
 import jwt from 'jsonwebtoken';
 import { NextPage } from 'next';
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
Personal preference, but I think it's easier to understand React hook usage when we _don't_ import hooks, and instead use them as `React.useState()` or `React.useEffect()`.  This allows us to distinguish between a custom hook (which will also be named as `useX`) and an official React hook. Non-blocking comment though!